### ISSUE TITLE
Casting crashes after a few seconds

### ISSUE THREAD

**Issue by gabriel-rreis** [benzea/gnome-screencast#95]
Hello,

I've been experiencing some issues after the last update I made on my system (Manjaro [Arch] GNOME). 

```
$ G_MESSAGES_DEBUG=all gnome-network-displays --display=$DISPLAY
(gnome-network-displays:25862): GLib-GIO-DEBUG: 17:22:01.488: _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
(gnome-network-displays:25862): Gtk-DEBUG: 17:22:01.574: Connecting to session manager
** (gnome-network-displays:25862): DEBUG: 17:22:01.635: NdScreencastPortal: Aquired Portal proxy
** (gnome-network-displays:25862): DEBUG: 17:22:01.635: task is: 0x557b8ed15010
** (gnome-network-displays:25862): DEBUG: 17:22:01.636: NdPulseaudio: Querying sink info by name
** (gnome-network-displays:25862): DEBUG: 17:22:01.636: NdPulseaudio: Error querying sink info
** (gnome-network-displays:25862): DEBUG: 17:22:01.636: NdPulseaudio: Got a sink info for the expected name
** (gnome-network-displays:25862): DEBUG: 17:22:01.639: WFDP2PRegistry: Found a new device, creating provider
** (gnome-network-displays:25862): DEBUG: 17:22:01.639: WfdP2PProvider: Discover is now set to 1
** (gnome-network-displays:25862): DEBUG: 17:22:01.642: WFDP2PRegistry: Got NMClient
** (gnome-network-displays:25862): DEBUG: 17:22:01.642: NdScreencastPortal: Received CreateSession response
** (gnome-network-displays:25862): DEBUG: 17:22:01.642: simple variant lookup: /org/freedesktop/portal/desktop/session/1_475/gtk930529
** (gnome-network-displays:25862): DEBUG: 17:22:01.643: NdScreencastPortal: Received SelectSource response
** (gnome-network-displays:25862): DEBUG: 17:22:04.212: NdScreencastPortal: Received Start response
** (gnome-network-displays:25862): DEBUG: 17:22:04.212: Got a stream with node ID: 15
** (gnome-network-displays:25862): DEBUG: 17:22:05.176: WFDP2PProvider: Found a new sink with peer 0x557b8eb06e10 on device 0x557b8e7fa220
** (gnome-network-displays:25862): DEBUG: 17:22:05.176: SinkList: Adding a sink
** (gnome-network-displays:25862): DEBUG: 17:22:05.177: WFDP2PProvider: Found a new sink with peer 0x557b8eb06f30 on device 0x557b8e7fa220
** (gnome-network-displays:25862): DEBUG: 17:22:05.177: SinkList: Adding a sink
** (gnome-network-displays:25862): DEBUG: 17:22:05.178: WFDP2PProvider: Found a new sink with peer 0x557b8ebdf0d0 on device 0x557b8e7fa220
** (gnome-network-displays:25862): DEBUG: 17:22:05.178: SinkList: Adding a sink
** (gnome-network-displays:25862): DEBUG: 17:22:05.855: WFDP2PProvider: Found a new sink with peer 0x557b8ebdf160 on device 0x557b8e7fa220
** (gnome-network-displays:25862): DEBUG: 17:22:05.855: SinkList: Adding a sink
** (gnome-network-displays:25862): DEBUG: 17:22:06.830: Found x264enc for video encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.830: Found fdkaacenc for audio encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.830: Found avenc_aac for audio encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.830: Found faac for audio encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.830: Got state change notification from streaming sink to state ND_SINK_STATE_WAIT_P2P
** (gnome-network-displays:25862): DEBUG: 17:22:06.830: WfdP2PProvider: Discover is now set to 0
** (gnome-network-displays:25862): DEBUG: 17:22:06.839: NdWfdP2PSink: Got P2P connection
** (gnome-network-displays:25862): DEBUG: 17:22:06.839: Found x264enc for video encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.839: Found fdkaacenc for audio encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.839: Found avenc_aac for audio encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.839: Found faac for audio encoding.
** (gnome-network-displays:25862): DEBUG: 17:22:06.839: Got state change notification from streaming sink to state ND_SINK_STATE_WAIT_SOCKET
** (gnome-network-displays:25862): DEBUG: 17:22:12.167: NdWfdP2PSink: Got client connection
** (gnome-network-displays:25862): DEBUG: 17:22:12.167: Got state change notification from streaming sink to state ND_SINK_STATE_WAIT_STREAMING
** (gnome-network-displays:25862): DEBUG: 17:22:12.696: WfdClient: OPTIONS querying done
** (gnome-network-displays:25862): DEBUG: 17:22:12.697: WFD query params
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: WfdClient: GET_PARAMS done
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: Add audio codec to params:
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: WfdAudioCodec: LPCM, 3, latency: 0
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: Add audio codec to params:
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: WfdAudioCodec: AAC, 1, latency: 0
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: Add audio codec to params:
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: WfdAudioCodec: AC3, 0, latency: 0
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: Add codec to params:
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: WfdVideoCodec:
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * profile: 2
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * level: 4
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * native: 1920x1080 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804: Supported resolutions:
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 800x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 800x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 854x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 854x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 864x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 864x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 640x360 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 640x360 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 960x540 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 960x540 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 848x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 848x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 640x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 720x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.804:  * 720x576 50p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x720 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x720 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1920x1080 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1290x720 25p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x720 50p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1920x1080 25p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x720 24p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1920x1080 25p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 800x600 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 800x600 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1024x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1024x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1152x864 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1152x864 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x800 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x800 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1360x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1360x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1366x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1366x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1280x1024 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1440x900 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1440x900 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1600x900 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1600x900 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1680x1024 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 1680x1050 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805: Add codec to params:
** (gnome-network-displays:25862): DEBUG: 17:22:12.805: WfdVideoCodec:
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * profile: 1
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * level: 4
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * native: 1920x1080 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805: Supported resolutions:
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 800x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 800x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 854x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 854x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 864x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 864x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 640x360 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.805:  * 640x360 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 960x540 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 960x540 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 848x480 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 848x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 640x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 720x480 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 720x576 50p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x720 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x720 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1920x1080 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1290x720 25p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x720 50p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1920x1080 25p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x720 24p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1920x1080 25p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 800x600 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 800x600 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1024x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1024x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1152x864 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1152x864 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x800 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x800 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1360x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1360x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1366x768 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1366x768 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1280x1024 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1440x900 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1440x900 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1600x900 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1600x900 60p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1680x1024 30p
** (gnome-network-displays:25862): DEBUG: 17:22:12.806:  * 1680x1050 30p

** (gnome-network-displays:25862): WARNING **: 17:22:12.806: WfdClient: No resolution found, falling back to standard FullHD resolution.
** (gnome-network-displays:25862): DEBUG: 17:22:12.806: selected resolution 1920, 1080 @30
** (gnome-network-displays:25862): DEBUG: 17:22:12.810: WfdClient: SET_PARAMS done
** (gnome-network-displays:25862): DEBUG: 17:22:12.813: WfdClient: Initialization done!
** (gnome-network-displays:25862): DEBUG: 17:22:12.813: use x11: 0
** (gnome-network-displays:25862): DEBUG: 17:22:12.841: WfdMedia init: Got 1 streams
** (gnome-network-displays:25862): DEBUG: 17:22:14.837: QOS: proportion: 0,500, processed: 1, dropped: 1
** (gnome-network-displays:25862): DEBUG: 17:22:14.843: QOS: proportion: 0,500, processed: 1, dropped: 2
** (gnome-network-displays:25862): DEBUG: 17:22:14.851: QOS: proportion: 0,500, processed: 1, dropped: 3
** (gnome-network-displays:25862): DEBUG: 17:22:14.858: QOS: proportion: 0,500, processed: 1, dropped: 4
** (gnome-network-displays:25862): DEBUG: 17:22:14.864: QOS: proportion: 0,500, processed: 1, dropped: 5
** (gnome-network-displays:25862): DEBUG: 17:22:14.871: QOS: proportion: 0,500, processed: 1, dropped: 6
** (gnome-network-displays:25862): DEBUG: 17:22:14.875: QOS: proportion: 0,500, processed: 1, dropped: 7
** (gnome-network-displays:25862): DEBUG: 17:22:14.879: QOS: proportion: 0,500, processed: 1, dropped: 8
** (gnome-network-displays:25862): DEBUG: 17:22:14.882: QOS: proportion: 0,500, processed: 1, dropped: 9
** (gnome-network-displays:25862): DEBUG: 17:22:14.886: QOS: proportion: 0,500, processed: 1, dropped: 10
** (gnome-network-displays:25862): DEBUG: 17:22:14.890: QOS: proportion: 0,500, processed: 1, dropped: 11
** (gnome-network-displays:25862): DEBUG: 17:22:14.897: QOS: proportion: 0,500, processed: 1, dropped: 12
** (gnome-network-displays:25862): DEBUG: 17:22:14.903: QOS: proportion: 0,500, processed: 1, dropped: 13
** (gnome-network-displays:25862): DEBUG: 17:22:14.908: QOS: proportion: 0,500, processed: 1, dropped: 14
** (gnome-network-displays:25862): DEBUG: 17:22:14.913: QOS: proportion: 0,500, processed: 1, dropped: 15
** (gnome-network-displays:25862): DEBUG: 17:22:14.918: QOS: proportion: 0,500, processed: 1, dropped: 16
** (gnome-network-displays:25862): DEBUG: 17:22:14.922: QOS: proportion: 0,500, processed: 1, dropped: 17
** (gnome-network-displays:25862): DEBUG: 17:22:14.924: QOS: proportion: 0,500, processed: 1, dropped: 18
** (gnome-network-displays:25862): DEBUG: 17:22:14.928: QOS: proportion: 0,500, processed: 1, dropped: 19
** (gnome-network-displays:25862): DEBUG: 17:22:14.932: QOS: proportion: 0,500, processed: 1, dropped: 20
** (gnome-network-displays:25862): DEBUG: 17:22:14.936: QOS: proportion: 0,500, processed: 1, dropped: 21
** (gnome-network-displays:25862): DEBUG: 17:22:14.940: QOS: proportion: 0,500, processed: 1, dropped: 22
** (gnome-network-displays:25862): DEBUG: 17:22:14.943: QOS: proportion: 0,500, processed: 1, dropped: 23
** (gnome-network-displays:25862): DEBUG: 17:22:14.946: QOS: proportion: 0,500, processed: 1, dropped: 24
** (gnome-network-displays:25862): DEBUG: 17:22:14.950: QOS: proportion: 0,500, processed: 1, dropped: 25
** (gnome-network-displays:25862): DEBUG: 17:22:14.953: QOS: proportion: 0,500, processed: 1, dropped: 26
** (gnome-network-displays:25862): DEBUG: 17:22:14.955: QOS: proportion: 0,500, processed: 1, dropped: 27
** (gnome-network-displays:25862): DEBUG: 17:22:14.957: QOS: proportion: 0,500, processed: 1, dropped: 28
** (gnome-network-displays:25862): DEBUG: 17:22:14.962: QOS: proportion: 0,500, processed: 1, dropped: 29
** (gnome-network-displays:25862): DEBUG: 17:22:14.963: QOS: proportion: 0,500, processed: 1, dropped: 30
** (gnome-network-displays:25862): DEBUG: 17:22:14.966: QOS: proportion: 0,500, processed: 1, dropped: 31
** (gnome-network-displays:25862): DEBUG: 17:22:14.969: QOS: proportion: 0,500, processed: 1, dropped: 32
** (gnome-network-displays:25862): DEBUG: 17:22:14.972: QOS: proportion: 0,500, processed: 1, dropped: 33
** (gnome-network-displays:25862): DEBUG: 17:22:14.973: QOS: proportion: 0,500, processed: 1, dropped: 34
** (gnome-network-displays:25862): DEBUG: 17:22:14.973: QOS: proportion: 0,500, processed: 1, dropped: 35
** (gnome-network-displays:25862): DEBUG: 17:22:14.974: QOS: proportion: 0,500, processed: 1, dropped: 36
** (gnome-network-displays:25862): DEBUG: 17:22:14.975: QOS: proportion: 0,500, processed: 1, dropped: 37
** (gnome-network-displays:25862): DEBUG: 17:22:14.976: QOS: proportion: 0,500, processed: 1, dropped: 38
** (gnome-network-displays:25862): DEBUG: 17:22:14.976: QOS: proportion: 0,500, processed: 1, dropped: 39
** (gnome-network-displays:25862): DEBUG: 17:22:14.976: QOS: proportion: 0,500, processed: 1, dropped: 40
** (gnome-network-displays:25862): DEBUG: 17:22:14.976: QOS: proportion: 0,500, processed: 1, dropped: 41
** (gnome-network-displays:25862): DEBUG: 17:22:14.977: QOS: proportion: 0,500, processed: 1, dropped: 42
** (gnome-network-displays:25862): DEBUG: 17:22:14.977: QOS: proportion: 0,500, processed: 1, dropped: 43
** (gnome-network-displays:25862): DEBUG: 17:22:14.983: QOS: proportion: 0,500, processed: 1, dropped: 44
** (gnome-network-displays:25862): DEBUG: 17:22:14.989: QOS: proportion: 0,500, processed: 1, dropped: 45
** (gnome-network-displays:25862): DEBUG: 17:22:14.998: QOS: proportion: 0,500, processed: 1, dropped: 46
** (gnome-network-displays:25862): DEBUG: 17:22:15.006: QOS: proportion: 0,500, processed: 1, dropped: 47
** (gnome-network-displays:25862): DEBUG: 17:22:15.008: QOS: proportion: 0,500, processed: 1, dropped: 48
** (gnome-network-displays:25862): DEBUG: 17:22:15.011: QOS: proportion: 0,500, processed: 1, dropped: 49
** (gnome-network-displays:25862): DEBUG: 17:22:15.014: QOS: proportion: 0,500, processed: 1, dropped: 50
** (gnome-network-displays:25862): DEBUG: 17:22:15.017: QOS: proportion: 0,500, processed: 1, dropped: 51
** (gnome-network-displays:25862): DEBUG: 17:22:15.018: QOS: proportion: 0,500, processed: 1, dropped: 52
** (gnome-network-displays:25862): DEBUG: 17:22:15.021: QOS: proportion: 0,500, processed: 1, dropped: 53
** (gnome-network-displays:25862): DEBUG: 17:22:15.024: QOS: proportion: 0,500, processed: 1, dropped: 54
** (gnome-network-displays:25862): DEBUG: 17:22:15.026: QOS: proportion: 0,500, processed: 1, dropped: 55
** (gnome-network-displays:25862): DEBUG: 17:22:15.030: QOS: proportion: 0,500, processed: 1, dropped: 56
** (gnome-network-displays:25862): DEBUG: 17:22:15.037: QOS: proportion: 0,500, processed: 1, dropped: 57
** (gnome-network-displays:25862): DEBUG: 17:22:15.039: QOS: proportion: 0,500, processed: 1, dropped: 58
** (gnome-network-displays:25862): DEBUG: 17:22:15.053: An audiocodec has been selected: yes
** (gnome-network-displays:25862): DEBUG: 17:22:21.468: NdWfdP2PSink: Got play request from client
** (gnome-network-displays:25862): DEBUG: 17:22:21.468: Got state change notification from streaming sink to state ND_SINK_STATE_STREAMING
Warning from pulseaudio-source: Can't record audio fast enough
** (gnome-network-displays:25862): DEBUG: 17:22:21.478: QOS: proportion: -71999989,541, processed: 21, dropped: 1
** (gnome-network-displays:25862): DEBUG: 17:22:21.626: QOS: proportion: -71999991,590, processed: 24, dropped: 17
** (gnome-network-displays:25862): DEBUG: 17:22:21.657: QOS: proportion: -71999996,946, processed: 23, dropped: 3
** (gnome-network-displays:25862): DEBUG: 17:22:21.685: QOS: proportion: -71999997,106, processed: 24, dropped: 4
** (gnome-network-displays:25862): DEBUG: 17:22:21.708: Forcing a keyframe!
** (gnome-network-displays:25862): DEBUG: 17:22:21.709: QOS: proportion: -71999997,148, processed: 25, dropped: 5
** (gnome-network-displays:25862): DEBUG: 17:22:21.738: QOS: proportion: -71999997,100, processed: 26, dropped: 6
** (gnome-network-displays:25862): DEBUG: 17:22:21.761: QOS: proportion: -71999997,366, processed: 27, dropped: 7
** (gnome-network-displays:25862): DEBUG: 17:22:21.790: QOS: proportion: -71999997,475, processed: 28, dropped: 8
Warning from pulseaudio-source: Can't record audio fast enough
Warning from pulseaudio-source: Can't record audio fast enough
Warning from pulseaudio-source: Can't record audio fast enough
Warning from pulseaudio-source: Can't record audio fast enough
** (gnome-network-displays:25862): DEBUG: 17:22:39.849: WfdClient: Doing keep-alive
** (gnome-network-displays:25862): DEBUG: 17:22:56.852: WfdMedia: Finalize
** (gnome-network-displays:25862): DEBUG: 17:23:09.465: WFDP2PProvider: Peer removed
** (gnome-network-displays:25862): DEBUG: 17:23:09.465: MetaSink: No usable sink is left, object has become invalid.
** (gnome-network-displays:25862): DEBUG: 17:23:09.465: SinkList: Removing a sink
** (gnome-network-displays:25862): DEBUG: 17:23:09.466: WFDP2PProvider: Peer removed
** (gnome-network-displays:25862): DEBUG: 17:23:09.466: MetaSink: No usable sink is left, object has become invalid.
** (gnome-network-displays:25862): DEBUG: 17:23:09.466: SinkList: Removing a sink
** (gnome-network-displays:25862): DEBUG: 17:23:09.466: WFDP2PProvider: Peer removed
** (gnome-network-displays:25862): DEBUG: 17:23:09.466: MetaSink: No usable sink is left, object has become invalid.
** (gnome-network-displays:25862): DEBUG: 17:23:09.466: SinkList: Removing a sink
** (gnome-network-displays:25862): DEBUG: 17:23:09.466: WFDP2PProvider: Peer removed
Segmentation fault (core dumped)
```

Seems like it is crashing on the keep-alive? Or is this some other package interfering with the application?

**Comment by benzea** [comment]
Well, there are two problems.

1. The keep-alive is likely not working. Sounds like issue #20.
2. It crashes on shutdown. Could be various issues; I have done related fixes.

Could you try the latest version from git? Or did you do that?

**Comment by benzea** [comment]
Closing as there was no new information.

Note that the fix in #103 might help you. Feel free to reopen if you still see issues with a the current development version.
